### PR TITLE
Help page

### PR DESCRIPTION
**Pull request by lxgr-linux**
Added a help page that's shown when pressing "?" to give the (new)player some information about the controls.
This should be part of v0.4.1

### CODE REVIEW

File: pokete.py [hunk 1]
```diff
@@ -1074,6 +1074,26 @@ def __call__(self, pokes):
...
+class Help(About):
+    def __init__(self, map):
+        self.map = map
+        self.help_text = """
+Controls:
+'w':up, 'a':left, 
+'s':down, 'd':right,
+'e':menu
+
+When walking into the hight grass (';') you may get attacked
```
Comment (MaFeLP) [inline pokete.py:1086]: Fix grammar:

```suggestion
When walking into the high grass (';') you may get attacked
```

File: pokete.py [hunk 2]
```diff
@@ -1074,6 +1074,26 @@ def __call__(self, pokes):
...
+by wild Poketes, those can be killed or weakened and caught.
+NPCs will talk to you when walking up to them.
+For more information about how to play this game, check out
+HowToPlay.md
```
Comment (MaFeLP) [inline pokete.py:1090]: Consider making this reference a link to this repo. People who don't know all the installation files or don't have a markdown viewer installed may not understand the file or what your reference means. 

Tipp: maybe look into GitHub pages to shorten the links. It is pretty easy and works with your wiki and the other files nicely 😉.

**Review by MaFeLP — CHANGES_REQUESTED** [review]

File: pokete.py (hunk 2 above)
Comment (lxgr-linux) [reply pokete.py:1090]: That's true, I will look into that.

Comment (lxgr-linux) [reply pokete.py]: Added the shortened link

**Review by MaFeLP — APPROVED** [review]

**Pull request merged**